conda_smithy/configure_feedstock.py

```diff
@@ -1354,20 +1369,35 @@ def _load_forge_config(forge_dir, exclusive_config_file):
         "azure": {
             # default choices for MS-hosted agents
             # for self-hosted agents, "vmImage" is replaced by "name" (of the pool)
-            "pool_linux": {"vmImage": "ubuntu-16.04"},
-            "pool_osx": {"vmImage": "macOS-10.14"},
-            "pool_win": {"vmImage": "vs2017-win2016"},
+            "settings_linux": {
+                "pool": {
+                    "vmImage": "ubuntu-16.04",
+                },
+                "timeoutInMinutes": 360,
+                "strategy": { "maxParallel": 8 }
+            },
+            "settings_osx": {
+                "pool": {
+                    "vmImage": "macOS-10.14",
+                },
+                "timeoutInMinutes": 360,
+                "strategy": { "maxParallel": 8 }
+            },
+            "settings_win": {
+                "pool": {
+                    "vmImage": "vs2017-win2016",
+                },
+                "timeoutInMinutes": 360,
+                "strategy": { "maxParallel": 4 },
+                "variables": { "CONDA_BLD_PATH": r"D:\\bld\\" }
+            },
             # disallow publication of azure artifacts for now.
             "upload_packages": False,
             # Force building all supported providers.
             "force": False,
             # name and id of azure project that the build pipeline is in
             "project_name": "feedstock-builds",
             "project_id": "84710dde-1620-425b-80d0-4cf5baca359d",
-            # Default to a timeout of 6 hours.  This is the maximum for azure by default
-            "timeout_minutes": 360,
-            # Could be overwritten for self-hosted agents; used when "matrix" is set.
-            "strategy": {"maxParallel": 8},
             # valid only when using self-hosted agents
             "workspace": {"clean": None},
```

Comment:
I think `workspace` should also be moved into `settings_*`, and be removed when MS-hosted agents are in use. Note that the acceptable values are "outputs", "resources",  and "all"; "None" is not (though I don't know if the agent would error out or not).